### PR TITLE
Moved footer and tryHoc banner inside wrapper so they share a container to fix scrolling bug.

### PR DESCRIPTION
**Pull request by aoby**
This bug is on prod now: 
screen shot 2015-12-04 at 10 32 11 am

The small footer is relative to the height of the window and behaves strangely with scroll bars.
Additionally, the tryHoc banner in https://github.com/code-dot-org/code-dot-org/pull/5850 pushes the footer down below the window. 

screen shot 2015-12-04 at 10 37 53 am
scroll down...
screen shot 2015-12-04 at 10 38 02 am

This fixes both so the footer is at the bottom of the page regardless of window height or presence of the banner.

### CODE REVIEW

**Comment by Bjvanminnen** [comment]
Just to make sure I understand the bug a little better - this is only an issue if we make our window smaller than the play space + buttons below it, right?

File: shared/js/npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+23 error Failed at the @cdo/shared@0.0.1 gulp script 'gulp'.
+23 error Make sure you have the latest version of node.js and npm installed.
+23 error If you do, this is most likely a problem with the @cdo/shared package,
+23 error not with npm itself.
+23 error Tell the author that this fails on your system:
+23 error     gulp
+23 error You can get their info via:
+23 error     npm owner ls @cdo/shared
+23 error There is likely additional logging output above.
+24 verbose exit [ 1, true ]
```
Comment (Bjvanminnen) [inline shared/js/npm-debug.log]: dont want to commit this file. i'd add npm-debug.log to your global .gitignore file

**Comment by Bjvanminnen** [comment]
Based on the code, these changes look okay. Ideally, I'd like to spend some time going through the test matrix for this change + your other one. Might have a chance to do this after our production bug bash.

**Comment by aoby** [comment]
That's correct for what's currently in production.

In https://github.com/code-dot-org/code-dot-org/pull/5850, the added tryHoc banner extends the page below the window by the same height. I might be able to fix that issue by just moving the banner inside the wrapper and leaving the footer alone, if we feel that's safer.

File: shared/js/npm-debug.log (hunk 1 above)
Comment (aoby) [inline shared/js/npm-debug.log]: ha, thanks

**Pull request merged**